comps/llms/vllm-openvino/README.md

```diff
@@ -0,0 +1,67 @@
+# Use vLLM with OpenVINO
+
+## Build Docker Image
+
+To build the docker image, run the command
+
+```bash
+bash ./build_vllm_openvino.sh
+```
+
+Once it successfully finishes you will have a `vllm:openvino` image. It can directly spawn a serving container with OpenAI API endpoint or you can work with it interactively via bash shell.
```

Comment:
s/Once it successfully finishes you will have a `vllm:openvino` image./Once it successfully builds, you will have the `vllm:openvino` image.

" It can directly spawn a serving container with OpenAI API endpoint" 
This sounds like it is a living thing with its own free will. 
How about instead ?
It can be used to spawn a serving container with an OpenAI API endpoint or you can work with it interactively via a bash shell.

Reply:
Done.